parteh/PRTAllometricCNPMod.F90

```diff
@@ -2079,37 +2079,40 @@ function GetNutrientTargetCNP(this,element_id,organ_id,stoich_mode) result(targe
     nc_repro    = this%bc_in(acnp_bc_in_id_nc_repro)%rval
     pc_repro    = this%bc_in(acnp_bc_in_id_pc_repro)%rval
     crown_damage = this%bc_in(acnp_bc_in_id_cdamage)%ival
-    
+
     ! Storage of nutrients are assumed to have different compartments than
     ! for carbon, and thus their targets are not associated with a tissue
     ! but is more represented as a fraction of the maximum amount of nutrient
     ! that can be bound in non-reproductive tissues
-    
+
     if(organ_id == store_organ) then
 
-       call bleaf(dbh,ipft,crown_damage,canopy_trim, elongf_leaf, leaf_c_target)
-       call bfineroot(dbh,ipft,canopy_trim,l2fr, elongf_fnrt, fnrt_c_target)
-       call bsap_allom(dbh,ipft,crown_damage,canopy_trim, elongf_stem, sapw_area,sapw_c_target)
-       call bagw_allom(dbh,ipft,crown_damage, elongf_stem, agw_c_target)
-       call bbgw_allom(dbh,ipft, elongf_stem, bgw_c_target)
-       call bdead_allom(agw_c_target,bgw_c_target, sapw_c_target, ipft, struct_c_target)
+      ! bug fix: Preventing decline in Carbon Flux During Leaf-Off Period in Deciduous PFTs
+      ! This call to organs' target biomass is independent of phenological stage 
+      ! more details: https://github.com/NGEET/fates/pull/1348
+
+      call bleaf(dbh,ipft,crown_damage,canopy_trim, 1.0_r8, leaf_c_target)
+      call bfineroot(dbh,ipft,canopy_trim,l2fr, 1.0_r8, fnrt_c_target)
+      call bsap_allom(dbh,ipft,crown_damage,canopy_trim, 1.0_r8, sapw_area,sapw_c_target)
+      call bagw_allom(dbh,ipft,crown_damage, 1.0_r8, agw_c_target)
+      call bbgw_allom(dbh,ipft, 1.0_r8, bgw_c_target)
 
```

Comment:
@sharma-bharat I think we still need the call to bdead_allom() .  The code that follows needs struct_c_target

Reply:
that was quick! got it